access-mgmt/README.adoc

```diff
@@ -0,0 +1,469 @@
+
+An ACL acts as a mapping of permitted operations on specified resources for a selected account or for all accounts in an organization. An account can be a user account or a service account. Operations correspond to Kafka APIs or request types that relate to the specified resource.
+
+For example, a `Read` operation for a `Topic` resource corresponds to the `Fetch`, `OffsetCommit`, and `TxnOffsetCommit` Kafka requests. A `Write` operation for a `Topic` resource corresponds to the `Produce` and `AddPartitionsToTxn` Kafka requests.
+
+The following table lists the supported ACL permissions in {product}.
+
+NOTE: The resource identifier `Is` supports the wildcard character `*` to denote any occurrences of the specified resource. For example, `Topic` = `is *` means any topic in a Kafka instance.
+
+.Supported ACL permissions
+[cols="25%,25%,25%,25%"]
+|===
+h|Resource type
+h|Resource identifier
+h|Access type
+h|Operations
+
+|`Consumer group`
+
+(For consumer group access to a resource)
+a|
+* `Starts with`
+* `Is`
+a|
+* `Allow`
+* `Deny`
+a|
+* `All`
+* `Read`
+* `Delete`
+* `Describe`
+
+|`Topic`
+
+(For access to a topic)
+a|
+* `Starts with`
+* `Is`
+a|
+* `Allow`
+* `Deny`
+a|
+* `All`
+* `Read`
+* `Write`
+* `Create`
+* `Delete`
+* `Alter`
+* `Alter configs`
+* `Describe`
+* `Describe configs`
+
+|`Transactional ID`
+
+(For producer access to a resource)
+a|
+* `Starts with`
+* `Is`
+a|
+* `Allow`
+* `Deny`
+a|
+* `All`
+* `Write`
+* `Describe`
+
+|`Kafka instance`
+
+(For access to Kafka instance permissions in ACLs)
+|None
+a|
+* `Allow`
+* `Deny`
+a|
+* `Alter`
+* `Describe`
+|===
+
+[role="_additional-resources"]
+.Additional resources
+https://kafka.apache.org/documentation/#security_authz_primitives[Authorization Primitives^] in Kafka documentation
+
+[id="proc-setting-permissions_{context}"]
+== Setting account permissions in a Kafka instance in {product}
+
+[role="_abstract"]
+In {product}, you can create Access Control Lists (ACLs) in your Kafka instances and set permissions for how other user accounts or service accounts can interact with an instance and its resources. You can manage access for only the Kafka instances that you create or for the instances that the owner has enabled you to access and alter.
+
+.Prerequisites
+* You've created a Kafka instance and the instance is in *Ready* state.
+* The user account or service account that you're setting permissions for has been created in the organization.
+
+.Procedure
+. In the {product} web console, go to *Streams for Apache Kafka* > *Kafka Instances* and click the name of the Kafka instance that you want to set permissions for.
+. Click the *Permissions* tab to view the current ACL permissions for this instance.
++
+--
+By default, new Kafka instances contain the following permissions. These permissions allow all accounts in the organization to view the instance permissions and to view topics in the instance, but not to produce or consume messages.
```

Comment:
This feels like it belongs in the earlier concept topic so it isn't making the reader scroll so much to the next step. Maybe move the details out, and keep the general note/reminder that all new instances have default permissions that allow view, but not produce and consume. (You could also put the brief reminder before the procedure, too - can't remember if the template allows for it after the abstract or prereqs).

Reply:
You're right! I had it here initially so newcomers would have an idea what to expect when they get there, but I agree. I've moved that bit up above in the supported ACL section right after the list of supported options, and it flows much better to say what's supported and then what's default there. And I just removed this altogether here from the proc, not needed imo. Thanks!